inspectit-oce-bootstrap/src/main/java/rocks/inspectit/oce/bootstrap/context/IInspectitContext.java

```diff
@@ -0,0 +1,34 @@
+package rocks.inspectit.oce.bootstrap.context;
+
+import java.util.Map;
+
+/**
+ * Interface abstraction for the InspectitContext.
+ */
+public interface IInspectitContext extends AutoCloseable {
+
```

Comment:
let's add Java doc for all the methods here, I guess this would also help in understanding the complete cycle of a context..